README.md

```diff
@@ -74,6 +74,81 @@ numbers = hashids.DecodeLong("KVO9yy1oO5j");
 `numbers` is now going to be:
 
     [ 666555444333222L ]
+    
+### Decoding a single id
+
+By default, Decode and DecodeLong will return an array. if you need to decode just one id you can use
+the following helper functions:
+
+```C#
+var hashids = new Hashids("this is my pepper");
+number = hashids.DecodeSingle("NkK9");
+```
+
+`number` is now going to be:
+
+    12345
+
+```C#
+var hashids = new Hashids("this is my pepper");
+int number;
+
+if (hashids.TryDecodeSingle("NkK9", out number)) { // Decoding hash successfull. }
+```
+
+`number` is now going to be:
+
+    12345
+
+You can handle the exception to see what went wrong with the decoding:
+
+```C#
+var hashids = new Hashids("this is my pepper");
+try
+{    
+    number = hashids.DecodeSingle("NkK9");
+}
+catch (NoResultException) { // Decoding the provided hash has not yielded any result. }
+catch (MultipleResultsException) { // The decoding process yielded more than one result when just one was expected. }
+```
+
+`number` is now going to be:
+
+    12345
+
+```C#
+var hashids = new Hashids("this is my pepper");
+number = hashids.DecodeSingleLong("KVO9yy1oO5j");
+```
+
+`number` is now going to be:
+
+    666555444333222L
+
+```C#
+var hashids = new Hashids("this is my pepper");
+long number;
```

Comment:
This is still not inlined

Reply:
@paulofaustino  By "inlining" I didn't mean writing everything in one line but putting variable declaration right in the out parameter. Please, indent the try/catch block as they were initially and change `TryDecodeSingle` and `TryDecodeSingleLong` to be:

```C#
if (hashids.TryDecodeSingle("NkK9", out int number))
{
    // Decoding hash successfull.
}
```

```C#
if (hashids.TryDecodeSingleLong("NkK9", out long number))
{
    // Decoding hash successfull.
}
```

Reply:
Sorry, I completely missed your sample up there and kept thinking about "one liner".